libcxx/src/tzdb.cpp

```diff
@@ -622,6 +623,36 @@ static void __parse_tzdata(tzdb& __db, __tz::__rules_storage_type& __rules, istr
   }
 }
 
+static void __parse_leap_seconds(vector<leap_second>& __leap_seconds, istream&& __input) {
```

Comment:
Could you please comment where we can find the specification for the thing your parsing?